### ISSUE TITLE
ENH: mixed effects / repeated measures

### ISSUE THREAD

**Issue by dengemann** [statsmodels/statsmodels#646]
How is this evolving in the moment? Looking at the sandbox there does not seem to be too much change.
I've got a couple of use cases for which I ideally need to report 'repeated measures' analysis
Also for quite some time I've been thinking about implementing a general solution applicable to to cognitive neuroscience / physiological data.
So typically it should handle scenarios like: Y ~ within_a x within_b x between_j. Y can be button press latencies, error rates, etc., or brain responses. In that case Y has to be estimated at location l,  time t (and frequency f).
I might give this a shot if no one is deeply devoted to this during the next weeks or so.

**Comment by yl565** [comment]
Is there anyone working on this currently? I happens to work a lot with mixed model with correlated errors and would like to give it a shot.

**Comment by kshedden** [comment]
Yes, this was merged a few years ago and is available as the MixedLM class.  Please try it out and open an issue if needed or ask a question at https://groups.google.com/forum/#!forum/pystatsmodels.

**Comment by yl565** [comment]
Hmm...Maybe I missed something but I thought the current MixedLM supports random effect but do not allow specifying error covariance structure, like the `correlation` in R lme
```R
lme(Y ~ X, random=~1 | id, correlation=corCompSymm(form=~1|id), method="ML", data=data)
```

**Comment by kshedden** [comment]
I don't know the lme syntax very well, but I believe the model you have specified using corCompSymm should be equivalent to a random intercept model, which we can certainly handle.  But we cannot handle time-series like dependence structures such as AR.  I would like to do that at some point but no time now (it is not in lme4 either AFAIK).

Our implementation is more similar to lme4 than to lme/nlme.  Again, AFAIK we can handle the full range of model specifications supported by lme4 (and even a few that I do not know how to specify in lme4).  Our vc syntax is quite general albeit somewhat harder to use.  Also, our code will run moderately/dramatically slower depending on the specifics.
